x-pack/plugins/security_solution/public/common/components/query_bar/index.tsx

```diff
@@ -141,34 +141,35 @@ export const QueryBar = memo<QueryBarComponentProps>(
     const timeHistory = useMemo(() => new TimeHistory(new Storage(localStorage)), []);
     const arrDataView = useMemo(() => (dataView != null ? [dataView] : []), [dataView]);
     return (
-      <SearchBar
-        showSubmitButton={false}
-        dateRangeFrom={dateRangeFrom}
-        dateRangeTo={dateRangeTo}
-        filters={filters}
-        indexPatterns={arrDataView}
-        isLoading={isLoading}
-        isRefreshPaused={isRefreshPaused}
-        query={query}
-        onClearSavedQuery={onClearSavedQuery}
-        onFiltersUpdated={onFiltersUpdated}
-        onQueryChange={onQueryChange}
-        onQuerySubmit={onQuerySubmit}
-        onSaved={onSavedQuery}
-        onSavedQueryUpdated={onSavedQueryUpdated}
-        refreshInterval={refreshInterval}
-        showAutoRefreshOnly={false}
-        showFilterBar={!hideSavedQuery}
-        showDatePicker={false}
-        showQueryInput={true}
-        showSaveQuery={true}
-        timeHistory={timeHistory}
-        dataTestSubj={dataTestSubj}
-        savedQuery={savedQuery}
-        displayStyle={displayStyle}
-        isDisabled={isDisabled}
-        hideTextBasedRunQueryLabel
-      />
+      <>
```

Comment:
Why the fragment?

Reply:
It was a left-over fragment.. Removing it now..